service/src/main/kotlin/fi/espoo/evaka/invoicing/data/FeeDecisionQueries.kt

```diff
@@ -106,6 +114,7 @@ val feeDecisionDetailedQueryBase =
         LEFT JOIN daycare ON part.placement_unit = daycare.id
         LEFT JOIN care_area ON daycare.care_area_id = care_area.id
         LEFT JOIN employee as approved_by ON decision.approved_by = approved_by.id
+        LEFT JOIN employee as finance_decision_manager ON daycare.finance_decision_manager = finance_decision_manager.id
```

Comment:
Instead of just picking whatever person is the decision handler at the time of the query, I think the person should be stored in a `fee_decision` row so that the data matches reality later as well.

Reply:
Oh, and before you change anything, the handler should be stored only for approved decisions. Drafts should still have it fetched dynamically. And since this is supposed to replace the area filter, the handler should be determined by the placement unit of the youngest child to match the behavior requested by the users. That is done so that decisions are split distinctly between handlers and the same person handles a family’s decisions for as long as possible.